tests/test_read.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+import pathlib
+import numpy as np
+
+import src.read as rd
+
+def test_preprocess():
+    """
```

Comment:
Expect these docstrings to change soon as we use a standard, more informative format for them.

Reply:
The docstrings have been implemented, in the recommended style.